linkedList/app/src/main/java/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,63 @@
+package linkedlist;
+
+public class LinkedList {
+    Node head = null;
+    Node tail = null;  // not strictly required
+
+    public LinkedList(){
+        this.head = null;
+        this.tail = null;
+    }
+
+
+    //Will insert at the end of the linkedList seemed more challenging
+    public void insert(int value)
+    {
+        Node trackedNode = new Node(value);
+
+        if(this.head == null){
+            this.head = trackedNode;
+        }else{
+            Node currentNode = head;
+            while (currentNode.next != null){
+                currentNode = currentNode.next;
+            }
+            currentNode.setNext(trackedNode);
+        }
+
+
+
+    }
+    public boolean includes(int value)
+    {
+
+        Node curr = this.head;
+
+        while (curr.next != null){
+            if(curr.value == value){
+                return true;
+            }
+            curr = curr.next;
+        }
+
+        return false;
+    }
+
+
+    public StringBuilder toStringA()
```

Comment:
This isn't quite what the assignment was looking for - it asked for a string containing the values in each node of the list. This creates a substring for each node and adds it to the string, but the substring does not contain the value of the node.